CMakeLists.txt

```diff
@@ -0,0 +1,109 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(backtrace)
+
+# Automake uses -frandom-seed initialized with file name of given file
+# but AFAIK it can't be done on CMake, so here's always same seed
+set(CMAKE_CXX_FLAGS "-DHAVE_CONFIG_H -funwind-tables -frandom-seed=mySeed -W -Wall -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes -Wold-style-definition -Wmissing-format-attribute -Wcast-qual -g -O2")
+set(CMAKE_C_FLAGS "-DHAVE_CONFIG_H -funwind-tables -frandom-seed=mySeed -W -Wall -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes -Wold-style-definition -Wmissing-format-attribute -Wcast-qual -g -O2")
+
+file(GLOB sources 

```

Comment:
Prefer explicit listing instead of glob.

Reply:
Why so? I find it tedious to add another files to list when my projects expands, when I can just run cmake again. Or is it because there is no directory structure which allows precise usage of wildcard, or is it because project is kinda mature and we don't expect it to grow across more files? Or is there other reason?

Reply:
that's fixed in latest commits